d2l-hm-constants-behavior.html

```diff
@@ -20,7 +20,8 @@
 			root: 'https://api.brightspace.com/rels/root',
 			thumbnailRegular: 'https://api.brightspace.com/rels/thumbnail#regular',
 			thumbnailSmall: 'https://api.brightspace.com/rels/thumbnail#small',
-			userProfile: 'https://api.brightspace.com/rels/user-profile'
```

Comment:
I agree there seem to be some inconsistencies with our rel naming and we could probably use some cleanup across the board. Don't think this is the PR for that work, though :)